### PR TITLE
[common-artifacts]Exclude from example creation.

### PR DESCRIPTION
**Pull request by struss**
Parent Issue : #4151
Draft PR : #4260

This commit add `UnidirectionalSequenceLSTM` to exclusion list using TC generation.

ONE-DCO-1.0-Signed-off-by: KiDeuk Bang <rrstrous@nate.com>

### CODE REVIEW

File: compiler/common-artifacts/exclude.lst [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 #[[ optimize : Exclude from circle optimization(circle2circle) ]]
 ## TensorFlowLiteRecipes
+optimize(UnidirectionalSequenceLSTM_000) # This will be removed after all luci components done.
```
Comment (mhs4670go) [inline compiler/common-artifacts/exclude.lst:8]: Does `UnidirectionalSequenceLSTM_000` contain things `circle2circle` cannot run like `is_variable`? AFAIK, it doesn't need to add it to `optimize` exclude list if related PRs are merged proper order.

Comment (struss) [reply compiler/common-artifacts/exclude.lst:8]: No, it is model that composed of all weight, bias and etc as input. If I add recipe at the last part, those line is not needed. I will remove that and add recipe on last part(after luci component add.)

**Review by mhs4670go — APPROVED** [review]
LGTM

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**